packages/credentials/src/presentation-exchange.ts

```diff
@@ -32,6 +32,12 @@ export class PresentationExchange {
   }): string[] {
     this.resetPex();
     const selectResults: SelectResults = this.pex.selectFrom(presentationDefinition, vcJwts);
+
+    // If errors exist in the results object the credentials provided didn't satisfy the requirements in the Presentation Definition
+    if(selectResults.errors?.length) {
```

Comment:
super nit: despite using JS for years i still get confused about falsy values. do you mind doing `!== 0`?